flow/core/params.py

```diff
@@ -29,9 +37,11 @@
 class TrafficLightParams:
     """Base traffic light.
 
-    This class is used to place traffic lights in the network and describe
-    the state of these traffic lights. In addition, this class supports
-    modifying the states of certain lights via TraCI.
+    This class is used to place customized traffic lights in the network
+    and describe the state of these traffic lights. In addition, this class
+    supports modifying the states of certain lights via TraCI. Lastly, this
+    class enables a baseline in which all nodes are set to either actuated
```

Comment:
nit: change word from baseline

Reply:
It is a baseline though, so Idk what to replace with?